src/components/presentational/Chat/Chat.css

```diff
@@ -0,0 +1,75 @@
+.mdhui-chat {
+    background: var(--mdhui-background-color-0);
+    font-size: 14px;
+}
+
+.mdhui-chat .mdhui-chat-log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    min-height: 20vh;
```

Comment:
I would remove the min-height, not much point?  Wondering how this will work on mobile and if this component should just stretch to fill the remainder of the screen and if the chat box should just be sticky on the bottom

Reply:
This needs some sort of min-height otherwise the chat window is completely empty at the beginning and it's not really clear that messages will start showing up in it. I thought the 20vh should work fine for both desktop and mobile?

Chat box sticky on the bottom is basically what the Popup is for.

Reply:
Hmm yeah, this might be worth chatting a bit further about how we would, say, incorporate this as a view builder component; what is the recommended UI paradigm etc.

I was not thinking that we would still necessarily even have the popup component; instead this AI Assistant could be shown on it's own entire view...not necessarily be floating on top of the current view; but instead using MyDataHelps.openApplication to open another view that is the AI Assistant.  (usually that is how you see "chat" applications work for instance)

In which case you'd want the text box anchored to the bottom while the user can scroll back through messages etc.

May be worth chatting live about this at some point if there's a good time though, (this might require some deep testing on mobile to deal with virtual keyboard stuff)